common_types/src/datum.rs

```diff
@@ -213,6 +213,7 @@ impl TryFrom<&SqlDataType> for DatumKind {
             SqlDataType::Int(_) => Ok(Self::Int32),
             SqlDataType::SmallInt(_) => Ok(Self::Int16),
             SqlDataType::String => Ok(Self::String),
+            SqlDataType::Varbinary(_) => Ok(Self::Varbinary),
             SqlDataType::Custom(objects, _) if objects.0.len() == 1 => {
                 match objects.0[0].value.as_str() {
                     "UINT64" | "uint64" => Ok(Self::UInt64),
```

Comment:
Shall we add some tests for these types?

Reply:
Good suggestion. I will add a test case that includes all types.